src/TSTransformer/util/types.ts

```diff
@@ -187,9 +188,9 @@ export function isEmptyStringType(type: ts.Type) {
 	return isStringType(type);
 }
 
-export function isRoactElementType(state: TransformState): TypeCheck {
+export function isJSXElementType(state: TransformState): TypeCheck {
```

Comment:
This is no longer used, so it can be removed - and all of `JSXSymbolManager` with it, as far as I can tell.